clients/drcachesim/common/trace_entry.h

```diff
@@ -406,6 +406,7 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_FILETYPE,
 
+    // Enum value == 10.
```

Comment:
Hmm, I usually just do `grep '^ *TRACE_MARKER_TYPE' clients/drcachesim/common/trace_entry.h | nl -v 0` to see the numbers.

Should we have the build process run that and emit the list into debug_memtrace.dox?
Not sure manually looking at this header is the best way to get the numeric values?

Reply:
I guess there's no downside since we won't change the order here.